### PR TITLE
do not always load bismillah

### PR DESCRIPTION
**Pull request by mmahalwy**

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32973

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32974

File: src/containers/Surah/index.js [hunk 1]
```diff
@@ -422,9 +422,16 @@ class Surah extends Component {
               }]
             }`
           }]}
-          style={[{
-            cssText: `.text-arabic{font-size: ${options.fontSize.arabic}rem;} .text-translation{font-size: ${options.fontSize.translation}rem;}` // eslint-disable-line max-len
-          }]}
+          style={[
+            {
+              cssText: `.text-arabic{font-size: ${options.fontSize.arabic}rem;} .text-translation{font-size: ${options.fontSize.translation}rem;}` // eslint-disable-line max-len
```
Comment (sabeurthabti) [inline src/containers/Surah/index.js]: Wasn't aware we are doing inline style(JSCSS).

Comment (mmahalwy) [inline src/containers/Surah/index.js]: It's a ghetto way of doing it :)

**Pull request merged**